### PR TITLE
docs(devtools): add docs for sdk go

### PR DESCRIPTION
**Pull request by bene2k1**
### Description

Added docs for Scaleway Go SDK

### CODE REVIEW

File: developer-tools/scaleway-sdk/go-sdk.mdx [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/scaleway/scaleway-sdk-go/utils"
```
Comment (SamyOubouaziz) [inline developer-tools/scaleway-sdk/go-sdk.mdx:49]: cant find the utils module when running the file. Is this up-to-date? cc @remyleone 🙏

File: developer-tools/scaleway-sdk/go-sdk.mdx [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+```go
+package main
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	scaleway "github.com/scaleway/scaleway-sdk-go/scw"
```
Comment (SamyOubouaziz) [inline developer-tools/scaleway-sdk/go-sdk.mdx:100]: I have 1 message per imported module saying:
> no required module provides package github.com/scaleway/scaleway-sdk-go/api/k8s/v1; to add it: go get github.com/scaleway/scaleway-sdk-go/api

Is it normal? I have to manually run 3 separate `go get` commands to retrieve the modules. cc @remyleone

File: developer-tools/scaleway-sdk/go-sdk.mdx [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+	k8s "github.com/scaleway/scaleway-sdk-go/api/k8s/v1"
+	vpc "github.com/scaleway/scaleway-sdk-go/api/vpc/v2"
```
Comment (SamyOubouaziz) [inline developer-tools/scaleway-sdk/go-sdk.mdx:102]: I have 1 message per imported module saying:
> no required module provides package github.com/scaleway/scaleway-sdk-go/api/k8s/v1; to add it: go get github.com/scaleway/scaleway-sdk-go/api

Is it normal? I have to manually run 3 separate `go get` commands to retrieve the modules. cc @remyleone

File: developer-tools/scaleway-sdk/go-sdk.mdx [hunk 4]
```diff
@@ -0,0 +1,213 @@
+---
+meta:
+  title: Scaleway Go software developement kit (SDK)
+  description: This page explains how to install and configure the Scaleway CLI locally
```
Comment (bene2k1) [inline developer-tools/scaleway-sdk/go-sdk.mdx:4]:
```suggestion
  description: This page explains how to install and use the Scaleway Go software developement kit (SDK)
```

File: developer-tools/scaleway-sdk/go-sdk.mdx [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+content:
+  h1: Scaleway Go software developement kit (SDK)
+  paragraph: This page explains how to install and configure the Scaleway CLI locally
```
Comment (bene2k1) [inline developer-tools/scaleway-sdk/go-sdk.mdx:7]:
```suggestion
  paragraph: This page explains how to install and use the Scaleway Python software developement kit (SDK)
```

File: developer-tools/scaleway-sdk/go-sdk.mdx [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+tags: devtools cli download-cli scaleway-cli command-line
```
Comment (bene2k1) [inline developer-tools/scaleway-sdk/go-sdk.mdx:8]:
```suggestion
tags: devtools sdk go-sdk scaleway-sdk go golang
```

**Review by ldecarvalho-doc — APPROVED** [review]

File: developer-tools/scaleway-sdk/go-sdk.mdx (hunk 3 above)
Comment (SamyOubouaziz) [reply developer-tools/scaleway-sdk/go-sdk.mdx:96]: It works now for some reason, probably missing GO dependencies

File: developer-tools/scaleway-sdk/go-sdk.mdx (hunk 1 above)
Comment (SamyOubouaziz) [reply developer-tools/scaleway-sdk/go-sdk.mdx:43]: Also, it works fine without "github.com/scaleway/scaleway-sdk-go/utils"

Comment (bene2k1) [reply developer-tools/scaleway-sdk/go-sdk.mdx:43]:
```suggestion
```

**Pull request merged**